optimum/intel/openvino/modeling.py

```diff
@@ -157,13 +157,13 @@ def forward(
         self.compile()
 
         inputs = {
-            "input_ids": input_ids,
-            "attention_mask": attention_mask,
+            "input_ids": input_ids.numpy(),
```

Comment:
It would be safer/more flexible to cast to numpy with `np.array(input_ids)` (and other tensors) because then inference will also work if the input data is already in Numpy format. It is not obvious that you would need to use PyTorch tensors for OpenVINO inference.

Reply:
I agree. Changed it for generative models as well.